services/movie.py

```diff
@@ -0,0 +1,46 @@
+from typing import Optional
+
+from django.db.models import Q, QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: Optional[list] = None,
+        actors_ids: Optional[list] = None
+) -> QuerySet:
+
+    if genres_ids and actors_ids:
+        return Movie.objects.filter(
+            Q(genres__pk__in=genres_ids)
+            & Q(actors__pk__in=actors_ids)
```

Comment:
You can remove this check if you put query into variable and than apply filters to this query if needed, eventually return this query